client/packages/coldchain/src/Equipment/ImportAsset/ImportReviewDataTable.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { FC, useState } from 'react';
+import {
+  DataTable,
+  Grid,
+  NothingHere,
+  Pagination,
+  SearchBar,
+  TooltipTextCell,
+  useColumns,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { ImportRow } from './EquipmentImportModal';
+
+interface ImportReviewDataTableProps {
+  importRows: ImportRow[];
+}
+export const ImportReviewDataTable: FC<ImportReviewDataTableProps> = ({
+  importRows,
+}) => {
+  const t = useTranslation('coldchain');
+  const [pagination, setPagination] = useState<Pagination>({
+    page: 0,
+    first: 10,
```

Comment:
`10` isn't a valid value for `rowsPerPage` and this causes the selection to fail. 
can change to `20` to make it work

Reply:
I'm curious why this is? Does it need to be higher? Or an enum in the back end?

Reply:
It's an enum - the display is a Select and if you don't provide one of the available options you'll get a console warning, and no value selected in the component.